apollo/apollo.go

```diff
@@ -168,12 +163,11 @@ func (c *client) ClientConfigParam(cpc *ConfigParamConfig, cfs ...CustomFunction
 //  3. NameSpace: select by user.
 //  4. ServerKey: {{.ServerServiceName}} by default.
 //     ClientKey: {{.ClientServiceName}}.{{.ServerServiceName}} by default.
-//  5. Cluster: DEFAULT_CLUSTER by default
-func (c *client) configParam(cpc *ConfigParamConfig, t *template.Template, cfs ...CustomFunction) (ConfigParam, error) {
+//  5. Cluster: default by default
+func (c *client) configParam(cpc *ConfigParamConfig, t *template.Template) (ConfigParam, error) {
 	param := ConfigParam{
 		Type:      JSON,
-		Content:   defaultContent,
-		NameSpace: cpc.Category,
+		nameSpace: cpc.Category,
```

Comment:
why not use namespaceID， the `namespaceID` seems useless now.

Reply:
It is used init config on `RegisterConfigCallback` function